CTP/shell/src/com/navercorp/cubridqa/shell/main/Context.java

```diff
@@ -114,8 +114,8 @@ public void reload() throws IOException{
 		this.envList = initEnvList(config);
 		this.toolHome = com.navercorp.cubridqa.common.CommonUtils.getEnvInFile (Constants.ENV_CTP_HOME_KEY);
 		
-		this.cleanTestCase = getProperty("main.testcase.clean", "false").equalsIgnoreCase("true");
-		this.isWindows = getProperty("main.testing.platform", "linux").equalsIgnoreCase("windows");
+		this.cleanTestCase = needCleanTestCase().equalsIgnoreCase("true") && !CommonUtils.isEmpty(getTestCaseBranch());
```

Comment:
It's better to remove getCleanTestCase() in Context.java.

Refactor as below:

public void reload() {
      ...
      this.cleanTestCase = getProperty("main.testcase.clean", "false").equalsIgnoreCase("true") && !CommonUtils.isEmpty(getTestCaseBranch());
}

public boolean needCleanTestCase()	{
	return this.cleanTestCase;
}